react/MuiCozyTheme/theme.js

```diff
@@ -431,12 +431,16 @@ normalTheme.overrides = {
   MuiDialogActions: {
     root: {
       margin: '16px 32px',
+      paddingBottom: 'env(safe-area-inset-bottom)',
       [normalTheme.breakpoints.down('sm')]: {
         margin: '8px 16px',
         '& button': {
           flexGrow: 1
         }
       },
+      '&.dialogActions-noSafeArea': {
+        paddingBottom: '0'
+      },
```

Comment:
Dans le commit c'est écrit FixedDialog should respect the safe area, là c'est mis sur tout les Dialog. Est ce qu'on devrait pas plutôt faire le contraire ? Le mettre que sur FixedDialog et ne pas le mettre sur les autres ?

Reply:
En relisant la conversation, je vois qu'en fait on veut l'avoir sur tout les Dialog. En fait c'est pas sur ConfirmDialog qu'on le veut pas, c'est sur n'importe quel Dialog qui n'est pas fullScreen non ? est ce qu'on devrait pas passer par la classe paperFullScreen plutôt que de faire une classe adhoc qu'on ne met que sur ConfirmDialog.